src/Microsoft.CodeQuality.Analyzers/UnitTests/QualityGuidelines/DoNotCallOverridableMethodsInConstructorsTests.cs

```diff
@@ -237,9 +224,16 @@ End Class
         }
 
         [Fact]
-        public void CA2214SpecialInheritanceCSharp()
+        public async Task CA2214SpecialInheritanceCSharp()
```

Comment:
💡 Can we break this test up into one test for System.Web and a second test for System.Windows.Forms?